### ISSUE TITLE
Alternative ways for importing source img

### ISSUE THREAD

**Issue by Stickan87** [DennisLiu1993/Fastest_Image_Pattern_Matching#6]
I have figured out how to export the results u get from the program to an external .txt file right now. Index,Position, score and angle.
Where in the code do I find how to set up a different url for the input images? Is there som way to add a button to "Get file...", and you specify what url that points to in the code?
In that way you could have , lets say a camera that saves a new file with the same filename in a specified url.

Just a thought, trying to figure this stuff out :)

The program itself works really great!

**Comment by DennisLiu1993** [comment]
at the left top, there is a button "file", letting you read source image and destination image

**Comment by Stickan87** [comment]
Yes, i noticed the "file" menu. Spent a few hours yesterday and figured out how to send a specified image-link directly when execute is pressed. 
My plan is to pass angle and position for pick-and place robot.

**Comment by DennisLiu1993** [comment]
you should transplant my code to your robot- control program so that you can get the angle and position

or using TCPIP or RS232 to communicate with this program
